app/scripts/controllers/transactions/lib/recipient-blacklist-checker.js

```diff
@@ -0,0 +1,36 @@
+const KeyringController = require('eth-keyring-controller')
+
+/** @module*/
+module.exports = {
+  checkAccount,
+}
+
+/**
+ * Checks if a specified account on a specified network is blacklisted.
+  @param networkId {number}
+  @param account {string}
+*/
+async function checkAccount (networkId, account) {
+
+  const mainnetId = 1
+  if (networkId !== mainnetId) {
+    return
+  }
+
+  const damnedMnemonic = 'candy maple cake sugar pudding cream honey rich smooth crumble sweet treat'
+  const keyringController = new KeyringController({})
+  const Keyring = keyringController.getKeyringClassForType('HD Key Tree')
+  const opts = {
+    mnemonic: damnedMnemonic,
+    numberOfAccounts: 10,
+  }
+
+  const accountToCheck = account.toLowerCase()
+  const keyring = new Keyring(opts)
+  const damnedAccounts = await keyring.getAccounts()
```

Comment:
Rather than hard-code a specific mnemonic, I would rather we simply have an array of blocked accounts, maybe kept in a separate `json` file for ease of reading.